packages/destination-actions/src/destinations/sendgrid/updateUserProfile/index.ts

```diff
@@ -0,0 +1,248 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { customFields, convertPayload } from '../sendgrid-properties'
+import { IntegrationError } from '@segment/actions-core'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'SendGrid Marketing Campaigns Update Contact',
```

Comment:
I would suggest we just call this "Update Contact" since the destination is already called SendGrid Marketing Campaigns.

```suggestion
  title: 'Update Contact',
```